src/snakemake/logging.py

```
@@ -743,6 +743,15 @@ def stop(self) -> None:
         if self.queue_listener is not None and self.queue_listener._thread is not None:
             self.queue_listener.stop()
 
+        # Flush and close plugin handlers managed by the QueueListener
+        # (not attached to the logger).
+        for handler in self.plugin_handlers:
+            try:
+                handler.flush()
+            except Exception:
+                pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)